arangodb-foxx-services/src/scripts/fs.d.ts

```diff
@@ -0,0 +1,337 @@
+/*
```

Comment:
@wajda is it typings for the fs module?

Reply:
if yes, I would recommend leaving @types/node in package.json and in the tsconfig.json
https://stackoverflow.com/questions/43048113/use-fs-in-typescript

Reply:
I tried that but unfortunately it didn't work. ArangoDB for some reason uses a slightly different [File system API](https://www.arangodb.com/docs/stable/appendix-java-script-modules-file-system.html).

> The implementation tries to follow the CommonJS [Filesystem/A/0](http://wiki.commonjs.org/wiki/Filesystem/A/0) specification where possible.

Not sure how many _specs_ for the FS module exist in the JavaScript world, but the one `@types/node` provides doesn't fit what ArangoDB is using. My intention for this type-def file was to replace it with a library one as soon as possible (either I find something existing, or make a PR to the `@types/arangodb`)